recipes/folly/all/patches/0016-find-packages-2.patch

```diff
@@ -0,0 +1,53 @@
+diff --git a/CMake/folly-deps.cmake b/CMake/folly-deps.cmake
+index 34e834d56..08f7712f7 100644
+--- a/CMake/folly-deps.cmake
++++ b/CMake/folly-deps.cmake
+@@ -52,21 +52,21 @@ find_package(DoubleConversion MODULE REQUIRED)
```

Comment:
Seems like this needs to be patched

https://github.com/conan-io/conan-center-index/blob/2236447915d07c7bb6db164013c1263a6ad48d96/recipes/folly/all/patches/0008-find-packages.patch#L9-L10

Conan has the correct behavior 

https://github.com/google/double-conversion/blob/32bc443c60c860eb6b4843533a614766d611172e/CMakeLists.txt#L89
https://github.com/google/double-conversion/blob/32bc443c60c860eb6b4843533a614766d611172e/CMakeLists.txt#L105

Reply:
Yes, seems like without this find_package was finding some system-wide installation. Thanks.